### DISCUSSION TITLE
I would like to ask if the index.html file can be configured after the build, and only index.js & index.css will be introduced?

### DISCUSSION THREAD

**Discussion by mengbo-ji** [web-infra-dev/rsbuild#3509] [Q&A]
![image](https://github.com/user-attachments/assets/933c00b7-57a2-46db-ba16-6b181ea42586)
目前打包后 会多一个9132这样的引入。
期望只有index.js和index.css干净的引入，好动态化

**Comment by 9aoy** [comment]
These chunks are generated according to the `splitChunks` configuration, and it seems that `all-in-one` meets your needs.

https://rsbuild.dev/zh/guide/optimization/split-chunk#all-in-one
